public/app/features/plugins/extensions/utils.tsx

```diff
@@ -91,6 +94,16 @@ export const wrapWithPluginContext = <T,>(pluginId: string, Component: React.Com
       return null;
     }
 
+    if (isSandboxEnabled) {
+      return (
+        <PluginContextProvider meta={pluginMeta}>
+          <div data-plugin-sandbox={pluginMeta.id}>
```

Comment:
my advice is to always return the wrapper div and only add the `data-plugin-sandbox` when the sandbox is on.

We might introduce side effects if we add an extra div based on conditions, better to always have the extra div.

Reply:
Good point, updated 👍